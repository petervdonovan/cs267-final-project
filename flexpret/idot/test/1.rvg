[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [get-hartid t0]
    [reduce-shmem +acc t0 0 a0 a1]
    [critical-hart a0 a2 [sim-print t0 a1]]
    [id die]}]]
]
