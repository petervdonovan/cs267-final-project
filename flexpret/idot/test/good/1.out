Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0:         28
Core-1:         28
Core-2:         28
Core-3:         28
Core-4:         28
Core-5:         28
Core-6:         28
Core-7:         28
Core-8:         28
Core-9:         28
Core-10:         28
Core-11:         28
Core-12:         28
Core-13:         28
Core-14:         28
Core-15:         28
Core-16:         28
Core-17:         28
Core-18:         28
Core-19:         28
Core-20:         28
Core-21:         28
Core-22:         28
Core-23:         28
Core-24:         28
Core-25:         28
Core-26:         28
Core-27:         28
Core-28:         28
Core-29:         28
Core-30:         28
Core-31:         28
Core-32:         28
Core-33:         28
Core-34:         28
Core-35:         28
Core-36:         28
Core-37:         28
Core-38:         28
Core-39:         28
Core-40:         28
Core-41:         28
Core-42:         28
Core-43:         28
Core-44:         28
Core-45:         28
Core-46:         28
Core-47:         28
Core-48:         28
Core-49:         28
Core-50:         28
Core-51:         28
Core-52:         28
Core-53:         28
Core-54:         28
Core-55:         28
Core-56:         28
Core-57:         28
Core-58:         28
Core-59:         28
Core-60:         28
Core-61:         28
Core-62:         28
Core-63:         28
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
Core-16 is done
Core-17 is done
Core-18 is done
Core-19 is done
Core-20 is done
Core-21 is done
Core-22 is done
Core-23 is done
Core-24 is done
Core-25 is done
Core-26 is done
Core-27 is done
Core-28 is done
Core-29 is done
Core-30 is done
Core-31 is done
Core-32 is done
Core-33 is done
Core-34 is done
Core-35 is done
Core-36 is done
Core-37 is done
Core-38 is done
Core-39 is done
Core-40 is done
Core-41 is done
Core-42 is done
Core-43 is done
Core-44 is done
Core-45 is done
Core-46 is done
Core-47 is done
Core-48 is done
Core-49 is done
Core-50 is done
Core-51 is done
Core-52 is done
Core-53 is done
Core-54 is done
Core-55 is done
Core-56 is done
Core-57 is done
Core-58 is done
Core-59 is done
Core-60 is done
Core-61 is done
Core-62 is done
Core-63 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
    at Top.scala:126 assert(false.B, "Program terminated sucessfully")
[1742] %Error: Top.sim.v:1944776: Assertion failed in TOP.Top
%Error: Top.sim.v:1944776: Verilog $stop
Aborting...
